uktrade/data-hub-frontend
# Reinstate focus indicator for Data Hub logo

## Description of change

One of the issues raised in the accessibility audit was that the Data Hub logo in the global header did not have a focus indicator. This has now been added.

I've also changed the 'Skip to main content' link to use React (using a `govuk-react` component) and removed some legacy code relating to the Nunjucks implementation of the header.

## Test instructions

Open this branch in Chrome and navigate to the global header using the tab key. When you get to the Data Hub logo the focus indicator should be visible (see screenshot).

## Screenshots

<img width="270" alt="Screenshot 2023-03-07 at 09 39 06" src="https://user-images.githubusercontent.com/36161814/223383872-2ea5d34f-38f7-45d1-980e-ad9bbfca4028.png">


## Checklist

[//]: # 'When submitting a PR make sure the code review guidelines have been satisfied.
https://github.com/uktrade/data-hub-frontend/blob/main/docs/Code%20review%20guidelines.md'
